pages/common/traceroute.md

```diff
@@ -18,6 +18,6 @@
 
 `traceroute -q 5 {{host}}`
 
-- Specify size in bytes of datagram:
+- Specify size in bytes of probing packet:
 
-`traceroute {{host}} {{size<=32768}}`
+`traceroute {{host}} 42`
```

Comment:
Good catch, @zkendall. Those should be changed too. You can either change it yourself in this PR, or we can merge as-is and change later, if you prefer.